src/validation.cpp

```diff
@@ -20,6 +20,7 @@
 #include "init.h"
 #include "policy/fees.h"
 #include "policy/policy.h"
+#include "policy/rbf.h"
```

Comment:
ATMP does policy-based decisions. I think the real answer here is ATMP belongs outside of validation.cpp, but dear god thats a big refactor (I think).